docs/getting-started/features.md

```diff
@@ -121,7 +121,7 @@ You may stop the Kubernetes cluster by quitting Rancher Desktop. You can close R
 
 **Resetting a Kubernetes cluster:**
 
-Refer [here](./getting-started/features.md#reset-kubernetesreset-kubernetes-and-container-images) to learn about resetting the Kubernetes cluster. Note that resetting will delete all workloads and configuration.
+Refer [here](#reset-kubernetesreset-kubernetes-and-container-images) to learn about resetting the Kubernetes cluster. Note that resetting will delete all workloads and configuration.
```

Comment:
Here the difference between current and your PR is absolute path vs relative path? Btn, the link works fine in the prod version as well as your PR.

Reply:
While the current version works, generally linking to sections within the current page is done directly with `#section`.

When doing a local preview using `yarn start` I was getting this error message: `Error: Docs markdown link couldn't be resolved: (./getting-started/features.md)`. Also, without fixing this one the subsequent error message about the other (actually invalid) link didn't show up.